browser/brave_features_internal_names.h

```diff
@@ -8,5 +8,7 @@
 
 constexpr char kBraveVPNFeatureInternalName[] = "brave-vpn";
 constexpr char kPlaylistFeatureInternalName[] = "playlist";
-
+#if BUILDFLAG(IS_WIN)
```

Comment:
sorry, didn't realize this was not already also in an `ENABLE_BRAVE_VPN` guard (kBraveVPNFeatureInternalName should be too)

Reply:
done